packages/frontend/src/hooks/toaster/useToaster.tsx

```diff
@@ -58,7 +58,7 @@ const useToaster = () => {
                     subtitle || action ? (
                         <Stack spacing="xs" align="flex-start">
                             {typeof subtitle == 'string' ? (
-                                <MDEditor.Markdown
+                                <MarkdownPreview
```

Comment:
`MarkdownEditor` uses `MarkdownPreview` to render markdown.
lot of react-query hooks use toasters so `MarkdownPreview` is mocked in tests by default